### ISSUE TITLE
Train on multiple gpus

### ISSUE THREAD

**Issue by quocanh010** [roboflow/rf-detr#17]
Hello, 

Thanks for the great work!  How could one train on multiple gpus with the current set up?

**Comment by quocanh010** [comment]
That works! Thanks!

**Comment by abhijeet00** [comment]
@isaacrob-roboflow 
When using Distributed Data Parallel (DDP) to train rf-detr on multiple GPUs, do we also need to scale the learning rate linearly with the number of GPUs?

Reference: https://github.com/Lightning-AI/pytorch-lightning/discussions/3706#discussioncomment-3960433
